README.md

```diff
@@ -1 +1,216 @@
-# Leaderboard
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <h3><b>Leaderboard</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Leaderboard\] ](#-Leaderboard-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Leaderboard] <a name="about-project"></a>
+
+**[Leaderboard]** is a javaScript project that allow users to add and remove scores.
+
+## 🛠 Built With <a name="built-with"></a>
+
+1. ✅ **HTML**
+2. ✅ **CSS**
+3. ✅ **JavaScript**
+4. ✅ **Webpack**
+5. ✅ **LightHouse**
+6. ✅ **WebHint**
+7. ✅ **Styelint**
+8. ✅ **ESLint**
+9. ✅ **Git**
+10. ✅ **Git flow**
+11. ✅ **Github**
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+-  **[Main page]**
+-  **[Responsive]**
+-  **[Good look and feel]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- ✅ You can see the live demo of this webpage by clicking: [Live Demo Link]()
```

Comment:
- [ ] Great job working on this milestone.
Since it is not a requirement to deploy the project at this time, you could either delete this section or simple use the text `Comong soon ...` Don't put a link that still redirects to your github repo.